2017_09_14/functions_for_fuzzy.py

```diff
@@ -0,0 +1,117 @@
+#!/usr/bin/env python2
+# -*- coding: utf-8 -*-
+"""
+Created on Wed Aug 30 16:51:30 2017
+
+@author: rach
+"""
+import numpy as np
+import os
+import sys
+import random 
+#import random
+cwd = os.getcwd()
+sys.path.insert(0, cwd + "/../../../Programming/Read_data/")
+sys.path.insert(0, cwd + "/../../../Programming/Geometry_Functions/")
+sys.path.insert(0, cwd + "/../../../Programming/Extra_functions/")
+from functions_for_winds import alphas_from_ratios, get_min_beta_vec, return_nearest_values, alphas_from_betas, get_min_beta, alphas_from_betas_with_min, mag_between_winds
+
+def sech(x):
+    return np.cosh(x)**(-1)
```

Comment:
why not `np.arccosh`?

Reply:
Also does numpy not define a sech function?